sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1281,9 +1289,113 @@ CleanupEventListFromResetCmdList(std::vector<pi_event> &EventListToCleanup,
   return PI_SUCCESS;
 }
 
-// Reset signalled command lists in the queue and put them to the cache of
-// command lists. A caller must not lock the queue mutex.
-pi_result resetCommandLists(pi_queue Queue) {
+/// @brief Cleanup events in the immediate lists of the queue.
+/// @param Queue Queue where events need to be cleaned up.
+/// @param QueueLocked Indicates if the queue mutex is locked by caller.
+/// @param QueueSynced Hint informing that queue was synchronized before the
```

Comment:
QueueSynced is not a hint, it is a fact, so document it as such.

Reply:
Thanks, fixed.